node/parachain/src/cli.rs

```diff
@@ -43,7 +60,7 @@ pub struct ExportGenesisStateCommand {
     pub output: Option<PathBuf>,
 
     /// Id of the parachain this state is for.
-    #[structopt(long, default_value = "200")]
+    #[structopt(long, default_value = "100")]
```

Comment:
do we add our parachain id already by default?

Reply:
No, but should we hardcode it to 35, since we don't plan to change it?